src/main/java/org/broadinstitute/hellbender/tools/exome/TargetArgumentCollection.java

```diff
@@ -31,7 +32,7 @@
  * </p>
  * <p>
  *   If even this mechanism does not provide a target file name, and the target file is deemed non-optional,
- *   the a {@link UserException.BadArgumentValue} is thrown.
+ *   the a {@link CommandLineException.BadArgumentValue} is thrown.
```

Comment:
Yes. As I mentioned above I tried it out after I made that comment and its harmless; javadoc can't resolve the link without being told where the target is since its not part of the GATK javadoc run, so it just doesn't emit a link for these.